pkg/R/RDD.R

```diff
@@ -841,6 +841,26 @@ setMethod("take",
             resList
           })
 
+#' First
+#'
+#' Return the first element of an RDD
+#'
+#' @rdname first
+#' @export
+#' @examples
+#'\dontrun{
+#' sc <- sparkR.init()
+#' rdd <- parallelize(sc, 1:10)
+#' first(rdd)
+#' }
+setGeneric("first", function(x) { standardGeneric("first") })
+
+setMethod("first",
+          signature(x = "RDD"),
+          function(x) {
+            unlist(take(x, 1))
```

Comment:
`unlist` doesn't guarantee that we return one element (it just converts a list to a vector). Using `[[1]]` makes sure that only the first element is returned.